phys/module_diagnostics_driver.F

```diff
@@ -224,6 +225,17 @@ SUBROUTINE diagnostics_driver ( grid, config_flags,                  &
           ! Scheme specific namelist inputs
             config_flags%cellcount_method,                     &
             config_flags%cldtop_adjustment,                    &
+         ! Dynamic_Lightning inputs
```

Comment:
@BarryHLynn 
Barry,
There is no conformity here, I agree. However, let's leave it better than you found it. Use the "&" spacing that you find above your commits. That's the best we can do.